ajo/personalization-hybrid/src/templates/index.handlebars

```diff
@@ -90,7 +90,8 @@
     </div>
   </div>
   <script>
-    alloy("applyResponse", {{safeString applyResponseParam}}).then(applyPersonalization("{{surface}}"));
+    const edgeResponse = JSON.parse(atob("{{safeString applyResponseParam}}"));
```

Comment:
☝️ here we base64decode the string and parse it as JSON before providing it to the `applyResponse` command.

in the browser it looks like this when you view source...

![screenshot-2024-01-25 at 17 03 09@2x](https://github.com/adobe/alloy-samples/assets/427213/48892400-4109-47af-bd09-880d77bd0245)